components/navigation.css

```
@@ -0,0 +1,23 @@
+.navigation {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  background-color: var(--color-white);
+  padding: 12px 29px;
```

I would suggest the following because it is a little more responsive to the different heights of the icons and unifies it better:
```suggestion
  padding: 10px 30px;
```